epoxy-processor/src/main/java/com/airbnb/epoxy/PoetExtensions.kt

```diff
@@ -102,6 +104,13 @@ private fun JavaClassName.getSimpleNamesInKotlin(): List<String> {
     return originalNames
 }
 
+// Does not support transferring complex annotations which
+// have parameters and values associated with them.
+fun JavaAnnotationSpec.toKPoet(): KotlinAnnotationSpec {
```

Comment:
maybe this can be nullable, and we would return null if the `JavaAnnotationSpec` has a code block?